core/include/keep.S

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014, STMicroelectronics International N.V.
```

Comment:
core/arch/arm/include/arm32_macros.S, core/arch/arm/include/arm64_macros.S, and lib/libutils/ext/include/asm.S exists because I couldn't get it working in a `.h` file. We have changed compiler a couple of times since then so perhaps that's fixed now.